actix-rt/CHANGES.md

```diff
@@ -1,14 +1,17 @@
 # Changes
 
-## [TBD] - [TBD]
+## [1.1.0] - 2020-04-08
 
-Added
-
-- Expose `System::is_set` to check if current system has ben started
+### Added
 
-- Add `Arbiter::is_running` to check if event loop is running
+* Expose `System::is_set` to check if current system has ben started [#124]
+* Add `Arbiter::is_running` to check if event loop is running [#99]
```

Comment:
These seams swapped. #99 added the `System::is_set` to the API and #124 added `Arbiter::is_running`.

Reply:
Good catch!

```suggestion
* Expose `System::is_set` to check if current system has ben started [#99]
* Add `Arbiter::is_running` to check if event loop is running [#124]
```